stream-chat-android-ui-components-sample/src/main/res/values-v23/themes.xml

```diff
@@ -12,7 +12,7 @@
     <style name="StreamTheme" parent="@style/StreamUiTheme">
         <item name="streamUiChannelActionsDialogStyle">@style/ChannelActionsDialogTheme</item>
     </style>
-
+    
```

Comment:
extra space

Reply:
Yep, this space makes this `themes.xml` file consistent with the others. We normally put one empty line between the styles.

Reply:
Oh okay - my bad then! :]